.github/workflows/wasp-lint.yml

```diff
@@ -0,0 +1,27 @@
+name: WASP Lint
+on:
+  push:
+permissions:
+  contents: read
+jobs:
+  golangci:
+    defaults:
+      run:
+        working-directory: wasp
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
@skudasov why not to use `paths` from github? https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore I think way the workflow will not be started at all

Reply:
We've decided to go with `dorny`'s action for now since GHA way of solving optional pipelines and required checks is not that good https://github.com/orgs/community/discussions/44490